docs/administration/content_organization/content_types.md

```diff
@@ -30,13 +30,66 @@ Each content type is characterized by a set of metadata which define the general
-
-**URL alias name pattern** – a pattern which controls how the virtual URLs of the Locations will be generated when content items are created based on this content type. Note that only the last part of the virtual URL is affected. The pattern works in the same way as the Content name pattern. Text outside the angle brackets will be converted using the selected method of URL transformation. If no pattern is provided, the system will automatically use the name of the content item itself. (Optional.)
+**Identifier** –
+an identifier for internal use in configuration files, templates, PHP code, etc.
+It must be unique, can only contain lowercase letters, digits, and underscores.
+The maximum length is 50 characters. (Mandatory.)
+
+**Description** –
+a detailed description of the content type. (Optional.)
+
+<a id="content-name-pattern"></a>
+**Content name pattern** –
+a pattern that defines what name a new content item based on this content type gets.
+The pattern usually consists of Field identifiers that tell the system which Fields it should use when generating the name of a content item.
+Each Field identifier has to be surrounded with angle brackets.
+Text outside the angle brackets is included literally.
+If no pattern is provided, the system will automatically use the first Field. (Optional.)
+
+??? note "Pattern examples"
+
+    The following pattern takes the value of the field with the identifier `title` (which is required):
+    ```
+    <title>
+    ```
+
+    The following pattern combines several field values:
+    ```
+    <firstname> <lastname>
+    ```
+
+    The following pattern takes the value of the field with the identifier `seo_title` (which is optional) if not empty,
+    else the value of the field with the identifier `short_title` (which is optional),
+    else it takes the one with identifier `title` (which is required):
+    ```
+    <seo_title|short_title|title>
+    ```
+
+    The following pattern takes the value of the field with the identifier `nickname` if not empty,
+    else it takes the ones with identifiers `firstname` and `lastname` with a space inbetween:
+    ```
+    <nickname|(<firstname> <lastname>)>
+    ```
+
+    - Input-output example:
+        - `fistname`: "*Alice*"
+        - `lastname`: "*Doe*"
+        - `nickname`: "" (empty)
+        - Generated content name: "*Alice Doe*"
+    - Input-output example:
+        - `fistname`: "*Robert*"
+        - `lastname`: "*Doe*"
+        - `nickname`: "*Bob*"
+        - Generated content name: "*Bob*"
+
+    **Notice that you won't be able to obtain a vertical bar `|`  or parentheses in the generated names as they are special characters.**
```

Comment:
```suggestion
    **The grouping`(`, `)` symbols used together are treated as special keywords and can only be used together as such, without the possibility to use them as standard characters. **
```

Nitpick: believe you can obtain them **in the generated names** - if the fields contain them, they just cannot be part of the pattern in other meaning than the "special one"

I did some tests and it looks like you can use the `|` symbol in its standard meaning - only the grouping is not allowed.

CT settings:
<img width="1333" alt="Zrzut ekranu 2024-10-9 o 13 57 21" src="https://github.com/user-attachments/assets/7168cf63-74a9-49e5-a642-f591a6e74c02">

Content:
<img width="879" alt="Zrzut ekranu 2024-10-9 o 13 57 29" src="https://github.com/user-attachments/assets/ce50af1a-3160-4333-9186-65f459ed5f40">

Reply:
@mnocon 
OK for removing pipe and being clearer about parentheses.
I'm adding an example of what could go wrong so one could even search for "EZMETAGROUP_0" and find this explanation. What do you think about the following phrasing?:

```suggestion
    **Notice that the grouping parentheses (characters `(`and `)`) are always treated as special characters, and can't be used as standard characters (unexpected text would appear in between like `(EZMETAGROUP_0)`).**
```